graph/model/models_gen.go

```diff
@@ -0,0 +1,23 @@
+// Code generated by github.com/99designs/gqlgen, DO NOT EDIT.
+
+package model
+
+type Job struct {
+	ID        int    `json:"id"`
+	CreatedAt string `json:"createdAt"`
+	UpdatedAt string `json:"updatedAt"`
+	Title     string `json:"title"`
+	Company   string `json:"company"`
+	Salary    string `json:"salary"`
+	Location  string `json:"location"`
+}
```

Comment:
Is there a way to reuse the model type that we have defined?

Reply:
This code is autogenerated, so I think it's best to keep the graphql schema's model separate.

Also we don't always want our api response to be exactly the same as our db models. E.g. delete status, password and other confidential or not required data for users